KAZARI_PLUGIN.md

```diff
@@ -0,0 +1,77 @@
+# Kazari
+
+Kazari is a JavaScript plugin developed on **Scala.Js** that allows to turn your project's existing documentation into interactive Scala worksheets. Basically it decorates your code snippets with extra functionality that allows users to run them in a [remote Scala evaluator](https://github.com/scala-exercises/evaluator) and edit it, by just including the script and some CSS files to your site.
+
+# Installation
+
+In order to use Kazari, you first need to include two scripts:
+
+* `sbt-microsites-js-opt.js`, containing the main code of the decorator plugin.
+* `sbt-microsites-js-jsdeps.js`, containing its javascript dependencies.
+
+Kazari is still in a WIP state so we haven't officially distributed it yet, but you can generate development versions following the steps you'll find in the "Building the plugin" section below.
+
+To allow users to edit code interactively, Kazari relies on a Javascript code editor called [CodeMirror](http://codemirror.net). CodeMirror comes integrated with the scripts you add to your documentation, but you'll need to include the following CSS files to your site:
+
+* `codemirror.css`, the main stylesheet used by CodeMirror.
+* Theme CSS: CodeMirror can be configured with different themes, as you'll see in just a minute. In order for the themes to work, you'll need to provide their associated stylesheet too (i.e.: `monokai.css` for the `Monokai` theme).
+
+Those files can be downloaded from the [CodeMirror official site](http://codemirror.net/doc/releases.html). Note that Kazari is compatible with CodeMirror version **5.19.0**.
+
+Once you get all your scripts and stylesheets ready, you just need to include them in your project like this:
+
+```html
+<body>
+
+<script type="text/javascript" src="../../../target/scala-2.11/sbt-microsites-js-jsdeps.js"></script>
+<script type="text/javascript" src="../../../target/scala-2.11/sbt-microsites-js-opt.js"></script>
+<link rel="stylesheet" href="codemirror.css">
+<link rel="stylesheet" href="monokai.css">
+
+<script type="text/javascript">
+	var config = kazari.model.PluginConfig("http://localhost:8080/eval", "auth_token_string", "monokai")
```

Comment:
Can we have only one call to a function for decorating the code? It would be more idiomatic JS.

```javascript
$(document).ready(function() {
    kazari.decorateCode("http://localhost:8080/eval", "auth_token_string", "monokai")
})	
```

If Scala.js doesn't allow this its fine as it is.